janusgraph-cql/pom.xml

```diff
@@ -0,0 +1,314 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.janusgraph</groupId>
+        <artifactId>janusgraph</artifactId>
+        <version>0.2.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>janusgraph-cql</artifactId>
+    <name>JanusGraph-CQL: Distributed Graph Database</name>
+    <url>http://janusgraph.org</url>
+
+    <properties>
+        <cassandra-driver.version>3.1.4</cassandra-driver.version>
+        <javaslang.version>2.0.5</javaslang.version>
+
+        <test.byteorderedpartitioner>byteorderedpartitioner</test.byteorderedpartitioner>
+        <test.murmur>murmur</test.murmur>
+        <test.murmur-ssl>murmur-ssl</test.murmur-ssl>
+        <test.jvm.opts>-Xms256m -Xmx1280m -ea -XX:+HeapDumpOnOutOfMemoryError -javaagent:${com.github.jbellis:jamm:jar}</test.jvm.opts>
+        <test.excluded.groups>org.janusgraph.testcategory.MemoryTests,org.janusgraph.testcategory.PerformanceTests,org.janusgraph.testcategory.BrittleTests</test.excluded.groups>
+    </properties>
+
+    <dependencies>
+        <!-- Compile Dependencies -->
+        <dependency>
+            <groupId>org.janusgraph</groupId>
+            <artifactId>janusgraph-core</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>com.datastax.cassandra</groupId>
+            <artifactId>cassandra-driver-core</artifactId>
+            <version>${cassandra-driver.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.javaslang</groupId>
```

Comment:
I've updated ticked #271 with a link to a branch in our repo with this migration. I'll create a PR once this is merged.